I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/act1.rpy", line 84, in script
    Pigsy "Leave her alone ya, freeloader."
Exception: Sayer 'Pigsy' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/act1.rpy", line 84, in script
    Pigsy "Leave her alone ya, freeloader."
  File "C:\Users\suriv\Downloads\My Things\IMPORTANT\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 678, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\suriv\Downloads\My Things\IMPORTANT\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 576, in eval_who
    raise Exception("Sayer '%s' is not defined." % who.encode("utf-8"))
Exception: Sayer 'Pigsy' is not defined.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Passion Fruits - Demo 1.0
Sun Aug 07 14:53:19 2022
